react/training/src/app/screens/Game/index.js

```diff
@@ -1,16 +1,18 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { handleClick, jumpTo } from '../../../redux/game/actions/action';
 
 import Board from './components/Board/index';
 import styles from './styles.scss';
 
 class Game extends Component {
-  state = { history: [{ squares: Array(9).fill(null) }], stepNumber: 0, xIsNext: true };
-
   getState = winner => {
     if (winner) {
       return `Winner: ${winner}`;
     }
-    return `Next player: ${this.state.xIsNext ? 'X' : 'O'}`;
+    return `Next player: ${this.props.xIsNext ? 'X' : 'O'}`;
```

Comment:
You can have something like this here `getState = winner => (winner ? `Winner: ${winner}` : `Next player: ${this.props.xIsNext ? 'X' : 'O'}`);`